internal/provider/resources/automation_test.go

```diff
@@ -323,9 +324,9 @@ func TestAccResource_automation(t *testing.T) {
 					testutils.ExpectKnownValueNumber(eventTriggerAutomationResourceNameAndPath, "trigger.event.within", 60),
 					testutils.ExpectKnownValue(eventTriggerAutomationResourceNameAndPath, "actions.0.type", "run-deployment"),
 					testutils.ExpectKnownValue(eventTriggerAutomationResourceNameAndPath, "actions.0.source", "selected"),
-					testutils.ExpectKnownValue(eventTriggerAutomationResourceNameAndPath, "actions.0.deployment_id", "123e4567-e89b-12d3-a456-426614174000"),
+					testutils.ExpectKnownValueNotNull(eventTriggerAutomationResourceNameAndPath, "actions.0.deployment_id"),
```

Comment:
The resource ID will be generated now that we're not referring to an existing deployment, so we just make sure the result isn't null.